typings/node/node.d.ts

```diff
@@ -1292,7 +1292,7 @@ declare module "crypto" {
         update(data: any): void;
         sign(private_key: string, output_format: string): string;
     }
-    export function createVerify(algorith: string): Verify;
+    export function createVerify(algorithm: string): Verify;
```

Comment:
this typing definition is not originated in this repo but comes from definitelytyped (https://github.com/DefinitelyTyped/tsd), may need to update type definition instead of update it.

Actually I noticed this definition is not being referenced in repo, maybe we can get rid of this.